src/modules/role/role.dto.ts

```diff
@@ -99,7 +99,8 @@ export class IssuerDTO implements Issuer {
   issuerType: string;
 
   @IsArray()
-  did: string[];
+  @IsOptional()
+  did?: string[];
```

Comment:
If my above comment is correct, do we even need the `did` property at all any more?
```suggestion
```

Reply:
I agree. Do we also need to update the `IssuerResolver.sol` or, for now, is it enough to remove the option of setting list of issuers on the front-end?

Reply:
Ah sorry, I think I was confused. This `did` property is on the role definition which hasn't changed (role definitions can indeed have a list of DIDs as the valid issuers. So I think we still need the `did` property.

I guess you're making it optional because role can either have a `did` array or a `roleName` as the issuer?

Reply:
Yes, I want role definition to conform to `IRoleDefinition`. May its better to import this interface from iam-contract. In order to do this though `enrollmentPrecondition`, `metadata` and `fields` are needs to be migrated as they are optional in cache-server. Better to do this in separate PR

Reply:
Ok, sounds good